packages/sdk/src/send/send-get-events.js

```diff
@@ -53,3 +62,22 @@ export async function sendGetEvents(ix, opts = {}) {
 
   return ret
 }
+
+export async function sendGetEvents(ix, opts = {}) {  
+  ix = await ix
+
+  invariant(
+    ix.events.start !== null || (
+      ix.events.blockIds !== null &&
+      Array.isArray(ix.events.blockIds) &&
+      ix.events.blockIds.length > 0
+    ),
```

Comment:
I find conditional blocks like this hard to mentally parse. 

I liked how you had assigned each case to a separate constant in the other PR -- that way it was clear what each case represents.

Also, is `ix.events.blockIds !== null` needed if `Array.isArray(ix.events.blockIds)` is also checked?

Reply:
Good catch! I updated this check top operate using some new descriptive variables.